FirebaseFunctionsSwift/Sources/Codable/Callable+Codable.swift

```diff
@@ -20,8 +20,10 @@ public extension Functions {
   /// Creates a reference to the Callable HTTPS trigger with the given name, the type of an `Encodable`
   /// request and the type of a `Decodable` response.
   /// - Parameter name: The name of the Callable HTTPS trigger
-  /// - Parameter requestType: The type of the `Encodable` entity to use for requests to this `Callable`
-  /// - Parameter responseType: The type of the `Decodable` entity to use for responses from this `Callable`
+  /// - Parameter requestAs: The type of the `Encodable` entity to use for requests to this `Callable`
+  /// - Parameter responseAs: The type of the `Decodable` entity to use for responses from this `Callable`
```

Comment:
Based on my understanding of [Swift argument labels vs parameter names](https://docs.swift.org/swift-book/LanguageGuide/Functions.html#:~:text=implicit%20return%20value.-,Function%20Argument%20Labels%20and%20Parameter%20Names,-Each%20function%20parameter), I think the before diff is the correct form because `requestType` is the parameter name and `requestAs` is the argument label. The `- Parameter` doc should therefore declare the parameter name. 

Xcode offers an opinion if you select the API signature and use the `option + cmd + / ` "add-docs" shortcut.

I'm not firm on this, but I remember wondering about it once and this ^ was the rationale that informed my decision then.

Reply:
I didn't know that one. TIL! Unfortunately it fails for this signature, but it works for me for others.  It turns out there is no need for parameter names for this one, so I removed them.